store/sqlstore/user_store.go

```diff
@@ -325,6 +325,24 @@ func (us SqlUserStore) UpdateMfaActive(userId string, active bool) error {
 	return nil
 }
 
+func (us SqlUserStore) GetMany(ids []string) ([]*model.User, error) {
```

Comment:
Didn't update the existing `Get` method since in a lot of places we rely on the error to be NoRowsErr.